test/NuGet.Core.Tests/NuGet.Packaging.Test/ManifestFileTest.cs

```diff
@@ -37,57 +37,60 @@ public void ManifestFileReturnsNoValidationResultIfSourceContainsWildCardCharact
         public void ManifestFileReturnsValidationResultIfSourceContainsInvalidCharacters()
         {
             // Arrange
-            var manifestFile = new ManifestFile { Source = @"bin\\|\\*.dll", Target = @"lib" };
+            char badChar = Path.GetInvalidPathChars()[0];
+            var manifestFile = new ManifestFile { Source = "bin" + Path.DirectorySeparatorChar + badChar + Path.DirectorySeparatorChar + "*.dll", Target = @"lib" };
```

Comment:
May be easier to see by using Replace like from your previous change:
    $@"bin\{badChar}\*.dll".Replace('\\', Path.DirectorySeparatorChar)